.meta/dietpi-imager

```diff
@@ -28,6 +30,9 @@
 	OUTPUT_IMG_NAME=''
 	OUTPUT_IMG_EXT='img'
 	GPT=0
+	CLONING_TOOL=''
+	CLONEZILLA_REPO='https://sourceforge.net/projects/clonezilla/files/clonezilla_live_alternative'
```

Comment:
@sal666 
What about the stable branch? What is the difference? Since alternative branch from version naming sounds like its for Ubuntu.

Reply:
Stable and testing Clonezilla live are based on Debian, alternative is based on Ubuntu. There are differences on hardware and UEFI secure boot support. Explanation is [here](https://drbl.org/faq/fine-print.php?path=./2_System/57_why_ubuntu_based_clonezilla_live.faq#57_why_ubuntu_based_clonezilla_live.faq).
Choose whatever version you like. Just changing this var should do the trick, the rest of the code is independent from the branch.

Reply:
@sal666 
Interesting, since Buster Debian actually has free UEFI secure boot support as well: https://packages.debian.org/buster/shim-signed
However no harm to leave it like this for now.